INDEX

```diff
@@ -112,6 +112,14 @@
     "official": false,
     "primary_pkg": "grafana6"
   },
+  "heimdall-dashboard": {
+    "MANIFEST": "heimdall-dashboard.json",
+    "description": "Heimdall Application Dashboard is an elegant solution to organize all your web applications and links onto a single page website.",
+    "icon": "https://icons.freenas.org/community-icons/heimdall-dashboard.png",
+    "name": "Heimdall",
+    "official": false,
+    "primary_pkg": "Git branch - Master"
```

Comment:
How does this primary_pkg with a custom string work? Shouldn't it be `null` in this case?

Reply:
Correct, if this isn't a legit package name, it should be set to "null"

Reply:
Thank you for the feedback; I'll fix this.